taskcluster/kinds/finetune-student/kind.yml

```diff
@@ -104,7 +104,7 @@ tasks:
                     finetune
                     {src_locale}
                     {trg_locale}
-                    $MOZ_FETCHES_DIR/corpus
+                    $MOZ_FETCHES_DIR/corpus.moses
```

Comment:
From my idea in another comment, `tok-*` would look like this here, which to me reads a bit more intuitively.

```suggestion
                    $MOZ_FETCHES_DIR/corpus.tok-moses
```